projects/storefrontstyles/scss/components/content/tab-paragraph-container/_tab-paragraph-container.scss

```diff
@@ -195,4 +195,8 @@ $useTabsOnPDP: false !default;
   } @else {
     @extend %pdpFlat !optional;
   }
+
+  .active {
+    background-color: var(--cx-color-inverse);
+  }
```

Comment:
Is that a fix in the olde styles?

Reply:
it is moved from `_order-detail-page.scss ` to here.